### PR TITLE
Script to download saved objects

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- Python script to download saved objects of different types for version control

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@avikganguly01 @fynmanoj

File: Kibana Visualisations/dashboard/Reporting .ndjson
Comment (fynmanoj) [inline Kibana Visualisations/dashboard/Reporting .ndjson:1]: there is a space in the file name. is that required?

File: Kibana Visualisations/export-saved-objects.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import requests
+import logging
+import os
+import json
+
+_logger = logging.getLogger(__name__)
+
+KIBANA_URL = "https://analytics.sandbox.fynarfin.io/"
```
Comment (fynmanoj) [inline Kibana Visualisations/export-saved-objects.py:8]: make this configurable from a yaml /properties file other than the code

File: Kibana Visualisations/export-saved-objects.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+]
+
+# Getting saved objects
+def get_saved_objects(type):
+    url = KIBANA_URL + "api/saved_objects/_find"
+
+    params = {"type": type}
+    headers = {
+        "kbn-xsrf": "true",
+        "Authorization": "Basic dW5kZWZpbmVkOiRLSUJBTkFQQVNT",
```
Comment (fynmanoj) [inline Kibana Visualisations/export-saved-objects.py:27]: make this configurable and, do not push credentials to github

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: Kibana Visualisations/dashboard/Reporting .ndjson
Comment (truthfool) [reply Kibana Visualisations/dashboard/Reporting .ndjson:1]: These are getting pulled from our saved objects from kibana. Nothing to do with code.

**Pull request merged**